bindata/cert-manager-deployment/cainjector/cert-manager-cainjector-deployment.yaml

```diff
@@ -18,12 +18,16 @@ spec:
       app.kubernetes.io/name: cainjector
   template:
     metadata:
+      annotations:
```

Comment:
Notes to my self,
Future TODO: test cainjector metrics, update our ServiceMonitor docs with these metric paths.